Update for MVAPICH MPI refresh

juhlee-microsoft opened the pull request:
1. Removed NBC test case
  - No allreduce needed
2. For manual investigation, allow the multiple times execution.
3. Moved two required dependency inside rhel8 only case
  - kernel-rpm-macro and dnf turned out only available in rhel 8 repo.
4. Applied the bug's workaround
5. Removed SubtestValues tag

vyadavmsft commented:
[LISAv2 Test Results Summary]
Test Run On           : 02/02/2019 00:24:39
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:16

   ID TestCaseName                                                                TestResult TestDuration(in minutes)
---------------------------------------------------------------------------------------------------------------------
    1 BVT-VERIFY-DEPLOYMENT-PROVISION                                                   PASS                 2.95
        FirstBoot : PASS
        FirstBoot : Call Trace Verification : PASS
        Reboot : PASS
        Reboot : Call Trace Verification : PASS

iamshital commented on Libraries/TestHelpers.psm1:
```diff
@@ -90,12 +90,7 @@ function Upload-RemoteFile($uploadTo, $port, $file, $username, $password, $usePr
-							else {
-								Write-Output "yes" | .\tools\pscp -v -pw $args[1] -q -P $args[2] $args[3] $username@${uploadTo}: ;
-							}
+							Write-Output "yes" | .\tools\pscp -scp -v -pw $args[1] -q -P $args[2] $args[3] $username@${uploadTo}: ;
```
@lizzha Agreed. This is core function. And needs to verified.

juhlee-microsoft replied:
@vyadavmsft please follow up the comment here and post the test result.

chvalean commented on Libraries/TestHelpers.psm1:
```diff
@@ -182,12 +177,7 @@ function Download-RemoteFile($downloadFrom, $downloadTo, $port, $file, $username
-				else {
-					Write-Output "yes" | .\tools\pscp.exe  -v -2 -unsafe -pw $password -q -P $port $username@${downloadFrom}:$testFile $downloadTo 2> $downloadStatusRandomFile;
-				}
+				Write-Output "yes" | .\tools\pscp.exe -scp -v -2 -unsafe -pw $password -q -P $port $username@${downloadFrom}:$testFile $downloadTo 2> $downloadStatusRandomFile;
```
let's remove the verbose flag from pscp, I don't think it's required and the information displayed are not relevant.

iamshital replied:
Actually it helps. If the download / upload is unsuccessful, it will show the reason for failure in verbose messages. Like - file not found / access denied etc. And it redirects all verbose messages to $downloadStatusRandomFile. The contents of this are only shown if the there any problem.
So let's keep it as it is.

vyadavmsft commented:
In my test pscp is not falling to scp when sftp fails.  in any case i think we  close this since we are trying to add support of sftp to ol.

juhlee-microsoft committed:
1. Removed NBC test case
  - No allreduce needed
2. For manual investigation, allow the multiple times execution.
3. Moved two required dependency inside rhel8 only case
  - kernel-rpm-macro and dnf turned out only available in rhel 8 repo.
4. Applied the bug's workaround
5. Removed SubtestValues tag

juhlee-microsoft committed:
Update SetupRDMA.sh

abhimarathe reviewed (approved):
Looks good to me.
Please merge if others approve.
